README.md

```diff
@@ -40,14 +54,13 @@
 ### Usage
 
 - You can download an extension for VSCode named "Live Server" so you can see the actual project working in the browser
-
+- Also, you can run npm I to install the dependencies and open it whit webpack
+- To run the program whit webpack you can write the command line `npm run start`
 
 ## Authors
 
 👤 **@BregornOriginal**
 
-- GitHub: [@githubhandle](https://github.com/@BregornOriginal)
-
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
```

Comment:
- [OPTIONAL] Please update your Readme, currently, some sections are missing such as  

``` ##  License ``` section with the MIT license at the end of your README file to make it more professional. I would also recommend you add this. 😄

Reply:
Ok @BenMukebo , Actually I wasn't using one License already. But I will add one for the next project and add it! Thank you for the suggestion! 💯